packages/next/src/build/utils.ts

```diff
@@ -1316,6 +1317,13 @@ export async function buildAppStaticPaths({
         dynamicIO,
         authInterrupts,
       },
+      waitUntil: undefined,
+      onClose: () => {
+        throw new InvariantError(
+          'unexpected onClose call in buildAppStaticPaths'
+        )
+      },
```

Comment:
this would've previously from inside `AfterContext` if used anyway, it's just that we had these params in a `Partial` so `onClose` wasn't required

Reply:
What does this error mean? What's the action when users see it?

Reply:
it's an `InvariantError`, so it adds "This is a bug in Next.js" at the end. the action is to file a GH issue.